Code/Framework/AzCore/Tests/Math/SfmtTests.cpp

```diff
@@ -107,6 +109,82 @@ namespace UnitTest
                 EXPECT_EQ(array64_2[i], g.Rand64());
             }
         }
+
+        template<size_t NumThreads, size_t NumResultsPerThread, typename ResultType>
+        bool ParallelTest()
+        {
+            Sfmt sfmt;
+
+            // Arbitrary but deterministic seed values to guarantee that we get the same random numbers produced every time.
+            AZ::u32 buffer[32] = { 0x8236ed73, 0x854aa369, 0xc7b68864, 0x5f1e49da, 0x58ea5a08, 0xa33fc74b, 0x0336bd81, 0x8d3c11ac,
+                                   0x7312bc57, 0x92fee3d1, 0x4b852f9f, 0xa7ac02ad, 0x4d72fb7a, 0x630641c6, 0x1edbeebf, 0xc18fdabe,
+                                   0xc6588dba, 0x6a821cf5, 0xec7e24b3, 0x44246d7e, 0x24af2f32, 0x4f64d44c, 0x44b24116, 0x65585572,
+                                   0x0f95038d, 0xd3e4c521, 0x6eea6bc1, 0x6d651ab5, 0x25dfc39e, 0x7502d183, 0xd32fc9bf, 0x9854093f };
```

Comment:
This buffer can be `constexpr`
```suggestion
            // Arbitrary but deterministic seed values to guarantee that we get the same random numbers produced every time.
            constexpr AZ::u32 buffer[32] = { 0x8236ed73, 0x854aa369, 0xc7b68864, 0x5f1e49da, 0x58ea5a08, 0xa33fc74b, 0x0336bd81, 0x8d3c11ac,
                                   0x7312bc57, 0x92fee3d1, 0x4b852f9f, 0xa7ac02ad, 0x4d72fb7a, 0x630641c6, 0x1edbeebf, 0xc18fdabe,
                                   0xc6588dba, 0x6a821cf5, 0xec7e24b3, 0x44246d7e, 0x24af2f32, 0x4f64d44c, 0x44b24116, 0x65585572,
                                   0x0f95038d, 0xd3e4c521, 0x6eea6bc1, 0x6d651ab5, 0x25dfc39e, 0x7502d183, 0xd32fc9bf, 0x9854093f };
```